editor/src/components/PropertiesPanel/PropertiesPanelDynamic.vue

```diff
@@ -361,7 +369,6 @@ const PropertiesPanel = defineComponent({
         });
       }
       this.generalData['data'] = data;
-      this.key++;
```

Comment:
This line was removed because it caused the sidebar to refresh 2 times and re-render the props unnecessarily. This was suggested and reviewed by @hjonathan.